### ISSUE TITLE
[Bug]: Installation freezes  Windows 10

### ISSUE THREAD

**Issue by glocalglocal** [Stirling-Tools/Stirling-PDF#2487]
### Installation Method

Local Installation

### The Problem

I tried to install `Stirling-PDF-win-installer.exe` (v0.36.3 on Windows 10) for the first time. Installation freezes at 90%. The popup displays the message: 

> Initializing Stirling-PDF
> Stirling-PDF
> Finalising startup...
> 90%

If I terminate and start `C:\Users\user\AppData\Local\Stirling-PDF\Stirling-PDF.exe` it tries to continue from where it stopped and freezes again. 

If I run the downloaded `Stirling-PDF-win-installer.exe` again, it offers to repair the installation, and in the end I get the same freeze at 90% as above. 

### Version of Stirling-PDF

0.36.3

### Last Working Version of Stirling-PDF

Docker and portable 0.36.0

### No Duplicate of the Issue

- [X] I have verified that there are no existing issues raised related to my problem.

**Comment by Ludy87** [comment]
Hi @glocalglocal can you please provide the `info.log` from the path `C:\Users\user\AppData\Local\Stirling-PDF\logs`?

**Comment by glocalglocal** [comment]
Upgraded to 0.36.4 and problem persists. `info.log` attached. 
[info.log](https://github.com/user-attachments/files/18178365/info.log)

**Comment by HerveKr** [comment]
Same for me .... Blocked at 90% on 2 Windows 11 Pcs

**Comment by Frooodle** [comment]
Are you running any anti viruses? 

I have tested this on 5 windows machines and it works on all
So trying to see what you have in common

Anti virus:
Admin/guest account: 
Changed install path yes/no

Whatever else you can think of that might differentiate you from other windows users

**Comment by HerveKr** [comment]
Use MalwareBytes and ad guard ...

**Comment by glocalglocal** [comment]
I use the Windows AV but only on-demand. I run as admin and haven't changed the default installation path. 

I don't see anything relevant in the Windows Events log. 

I have the docker version installed and the portable version. Both work fine but the docker version won't work if I use the default port 8080. 8081, 8082 work OK. 

The docker version sees the applications for conversion to docx and compression. The install and portable versions don't, but I am guessing that's by design.  

I don't see anything unusual about my setup.

**Comment by Frooodle** [comment]
https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.36.5/Stirling-PDF-test.exe
Can you see if this version results in anything extra? i have added some logs

**Comment by Frooodle** [comment]
You will need to uninstall the old version first (should show up in add/remove apps/programs on windows)

**Comment by HerveKr** [comment]
Yesss this one is one , continue after 90% and see 
![image](https://github.com/user-attachments/assets/706edf1e-6fda-4c27-98f6-4819e08cfb47)

**Comment by Frooodle** [comment]
very odd... i didnt add anything to fix this! I only added more logs to investigate the issue.... 

Will remain a mystery.... Well will push this in next release anyway :) thanks!

**Comment by HerveKr** [comment]
lol

**Comment by Frooodle** [comment]
Perhaps the issue is resolved by uninstalling and reinstalling somehow

**Comment by HerveKr** [comment]
Tested on a second PC : blocked yesterday and uninstall yesterday : 
On this too , it's Ok Now (frist open in admin, Ok and after in user mode Ok too

Dont know why lol

**Comment by HerveKr** [comment]
For information : this morning tested on a 3nd PC
Classical exe : ask me for firewall windows (local and not locally) , accept , and finally after install block at 90% 
Test version : dont ask me for firewall , and directly Ok 
Perhaps something to see with firewall

**Comment by glocalglocal** [comment]
The test version completed OK here too. If there is nothing substantively different with this, then there is still an issue that is hit and miss. [HerveKr](https://github.com/HerveKr) and I were affected once and reported it. Others may not report it.

I see the install version doesn't do compress and convert to docx. If that's how it is designed to be, I will have to go back to the full docker version anyway which uses ghostscript and LibreOffice for these features.

**Comment by BloodyKitchen** [comment]
I got the same situation during installation,initializing stuck in 90% even if I uninstall and reinstall can't resolve this problem,and I using the test version it finally work out.

**Comment by pehamu** [comment]
I had the same issue with the regular Windows installer on a Win 11 laptop. Uninstalled, rebooted, reinstalled, same problem. Tried the test version... installation completed, but then when I started up the program, I get an alert that says "Failed to launch JVM"...
Hope someone can help. Thanks!
![Screenshot (1)](https://github.com/user-attachments/assets/be985623-55d2-4148-81e3-a0e1dee58f6d)

**Comment by Frooodle** [comment]
@pehamu  any extra info in logs folder 
 from the path C:\Users\user\AppData\Local\Stirling-PDF\logs?
 
 Also can you open CMD and run ``java --version`` just for curiosity

**Comment by pehamu** [comment]
Thanks Anthony!

Attached are the logs.

And the CMD prompt gave me this:
C:\Users\Peter Murray>java --version

'java' is not recognized as an internal or external command,

operable program or batch file.

Best,
Peter

**Comment by Frooodle** [comment]
Seems logs didn't attach via the GitHub email reply

**Comment by pehamu** [comment]
> Seems logs didn't attach via the GitHub email reply

[info.log](https://github.com/user-attachments/files/18242591/info.log)
[invalid-auths.log](https://github.com/user-attachments/files/18242592/invalid-auths.log)

**Comment by Flithor** [comment]
I was also stuck at 90%.
**But can use it by open http://localhost:8080 in browser.*
more info: The log file is locked by progress when running(Stirling-PDF.exe, jcef_helper.exe)
logs files:
[logs.zip](https://github.com/user-attachments/files/18278746/logs.zip)

**Comment by greaturchin** [comment]
I had the same issue where it stuck at 90%. I came here, read all the posts, put my computer aside for the evening, came to the laptop the next day and it was finished. Not sure when it completed, but it eventually did on its own. Hopefully that provides something.

**Comment by alexgan123** [comment]
I installed Stirling PDF from https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.36.5/Stirling-PDF-win-installer.exe and when I open the app, it is also stuck on finalising startup at 90%.

Here is my logs:
[info.log](https://github.com/user-attachments/files/18279178/info.log)

**Comment by topoldo** [comment]
Updating from 0.34 to 0.36.5, running with admin permission and using the "official" version..
Putting the executable in:
C:\stirlingtools\Stirling-PDF
Opened the firewall both for private & public networks
Installation freeze at 90%
info.log and debug.log attached
I'll also try with the snapshot on next day.

Topoldo

[debug.log](https://github.com/user-attachments/files/18290407/debug.log)
[info.log](https://github.com/user-attachments/files/18290412/info.log)


Topoldo

**Comment by topoldo** [comment]
I forgot: I have Win11 24H2

**Comment by topoldo** [comment]
Hi.
I uninstalled my Stirling.pdf version **0.34** leaving the original subdir:
C:\Program Files\stirlingtools\configs
which contained 0.34 version of: 
- custom_settings.yml (0 byte)
- settings.yml
The content of this original 0.34 **settings.yml** is herein attached (with a **.txt** extension because of limitation on kinds of attachment of this forum):

[settings.yml.txt](https://github.com/user-attachments/files/18291322/settings.yml.txt)

Then I downloaded the "**test**" version:
https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.36.5/Stirling-PDF-test.exe

and I tried to install it into two different locations, running in both cases the installation and the program using **admin permissions**.

In these conditions **I was successfull in both cases** and herein I attach the logs of both kinds of installation:

1. _C:\Users\Leopoldo\AppData\Local_
[debug_local.log](https://github.com/user-attachments/files/18291255/debug_local.log)
[info_local.log](https://github.com/user-attachments/files/18291257/info_local.log)

2. _C:\Program Files\stirlingtools\_
[debug_prg.log](https://github.com/user-attachments/files/18291259/debug_prg.log)
[info_prg.log](https://github.com/user-attachments/files/18291261/info_prg.log)

Regards,
Leopoldo/Topoldo

**Comment by Frooodle** [comment]
@topoldo  Do you believe the fix was admin permissions or something else?

**Comment by topoldo** [comment]
No idea, sorry. I don't know which are the differences between the "**test**" and "**official**" version and I also don't know if the problem is still there using 0.36.6 version.
In fact I decided to move to the windows-server version (0.35.5 > 0.35.6) which were both ok and don't need all the stuffs required by the standalone version, at least if I run them without:
- Convert file to PDF
- Detect/Split Scanned photos

features, as I described here:
[https://github.com/Stirling-Tools/Stirling-PDF/issues/2588](url)

In fact I don't know if I need to dowload and install python and/or other stuffs to be able to install/run **unoconv** and/or **OpenCV**

Regards,
Leopoldo/Topoldo

**Comment by Frooodle** [comment]
For the people still facing issues with windows installs can someone please try 
https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.36.6/Stirling-PDF-test.exe
and see if it fixes OR produces any different logs?

**Comment by topoldo** [comment]
I tried to install **Stirling-PDF 0.36.6** as "_standalone_" (not in server mode) using the "_windows-installer_" **0.36.6** version and I wasn't successfull: it still stops at 90% as the previous version.
So I moved to the _test_ one, same version. Unfortunately I wasn't successfull also in this case  :-(

Please note that, as I want the program to be accessible to all the users and, for a personal style in maintaning programs, I want all the programs go to
C:\Program Files\
or
C:\Program Files (x86)\
I tried to install both official and test version in:
C:\Program Files\stirlingtools
So I had to use the **admin** permissions in both cases because the program must "write" in this directory to create config files etc...

Besides, before launching Stirling PDF, I also changed the properties of the program shortcut on the desktop to make it to run as admin to be sure that no permission problem could interfere with its work.

PS:  BTW, IMHO, it could be better if all the Stirling pdf subdir were put in %APPDATA% (local or roaming) because data and configs maybe different for different users, while the executable should go in %PROGRAM FILES%

Regards,
Topoldo

**Comment by Frooodle** [comment]
Already working on the log/config file movement :) 
But that's separate to this 90% issue

**Comment by Frooodle** [comment]
Thanks for checking again on the installer side

**Comment by greaturchin** [comment]
It still stops at 90% for me

![image](https://github.com/user-attachments/assets/92c31867-69de-4cb2-9aca-538ae1a8e2d4)

**Comment by Frooodle** [comment]
Can you check AppData\Roaming\Stirling-PDF for logs. And also see if any log  file in webui folder

**Comment by topoldo** [comment]
Hi.
In my **Win11 24H2**, I removed all my previous installations of StirlingPDF: both as a **server** (no need to uninstall - just removed it and all its subfolders from the PC) and the "**standalone**" version - in this case by using the uninstall feature of the Windows Control Panel.

Then I installed "test3" version as **standard user**.
Also in these condition I had a freezing at 90%.
The info.log which I found in:
%APPDATA%\Roaming\Stirling-PDF\logs\
was this:
[info1.log](https://github.com/user-attachments/files/18307570/info1.log)

At this point I killed the StirlingPDF process and re-launched StirlingPDF as **admin** from its desktop shortcut.
However also in this case it froze at 90%.
In this case the log which was the "sum" of the previous log + lines added by the second execution is herein attached:
[info2.log](https://github.com/user-attachments/files/18307578/info2.log)

Seeing that last line of this log was different from the last one of the first execution, and that it was: 
`2025-01-04 17:38:27,489 INFO s.s.S.SPdfApplication [main] Navigate to http://localhost:8080`

I tried to open Firefox (which is my _default_ browser) and I left it opened, while I re-launched (as _admin_) Stirling-PDF for the third time using its Desktop shortcut.
Unfortunately also in this case the program froze at 90% and its log (which is the combination of the 2 previous logs + the lines added by this last attempt) is reported in the attachment below:
[info3.log](https://github.com/user-attachments/files/18307590/info3.log)

HTH
Topoldo

**Comment by djc0739** [comment]
This time I tried the new download on a Win 11 machine with the same results. Had to allow Windows Defender block and installation stopped at 90%. Please see attached.
[info.log](https://github.com/user-attachments/files/18307652/info.log)

**Comment by Frooodle** [comment]
I wonder what your installations all have in common
If its not OS etc is it locale?

What default language etc is installed for everyone?

**Comment by djc0739** [comment]
Both my installs were on Dell notebooks. The Win 10 machine is at 4 or 5 years old and the Win 11 machine is less than a year old.
My locale is English, USA, Eastern Time Zone.

**Comment by Frooodle** [comment]
what CPUs are they?

**Comment by Ludy87** [comment]
## freezes

| x | x |
|---|---|
| Prozessor | Intel(R) Celeron(R) CPU  N2940  @ 1.83GHz   1.83 GHz |
| Installierter RAM | 4,00 GB (3,88 GB verwendbar) |
| Systemtyp | 64-Bit-Betriebssystem, x64-basierter Prozessor |
| Edition	| Windows 10 Home |
| Version	| 22H2 |
| Betriebssystembuild	| 19045.4170 |
| Leistung	| Windows Feature Experience Pack 1000.19054.1000.0 |


## does not freeze

| x | x |
|---|---|
| Prozessor	| AMD Ryzen 7 3700U with Radeon Vega Mobile Gfx     2.30 GHz |
| Installierter RAM	| 40,0 GB (37,9 GB verwendbar) |
| Systemtyp	| 64-Bit-Betriebssystem, x64-basierter Prozessor |
| Edition	| Windows 11 Home |
| Version	| 23H2 |
| Betriebssystembuild	| 22631.4602 |
| Leistung	| Windows Feature Experience Pack 1000.22700.1055.0 |

**Comment by Frooodle** [comment]
Not working on
Processor	Intel(R) Core(TM) Ultra 9 185H 
Version	10.0.22631 Build 22631

**Comment by topoldo** [comment]
### Freezes

![Immagine 2025-01-05 093500](https://github.com/user-attachments/assets/1118222d-d306-4ac6-9899-feb20563b9cb)

Later (I don't know if I can today or tomorrow) I could test 2 more devices with old processors
Topoldo

**Comment by HerveKr** [comment]
Freeze too with this one :

**Comment by djc0739** [comment]
My Win 10 machine is: Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz, 1992 Mhz, 4 Core(s), 8 Logical Processor.
My Win 11 machine is: 12th Gen Intel(R) Core(TM) i7-1255U CPU 1700 Mhz, 10 Core(s), 12 Logical Processor.

**Comment by Frooodle** [comment]
Weirdly i managed to resolve the loading issue by getting it to 90%
Put PC to sleep
re-wake laptop and then it loads, no idea what this means the issue is, likely some thread issue, we will keep investigating

**Comment by Flithor** [comment]
@Frooodle Nice try, I found that too, but I can use on browser so I'm not consider any more.

**Comment by Strongly7402** [comment]
Same issue with v0.36.6. Installation worked fine by running Stirling-PDF-win-installer.exe. App launch freezes at 90% saying "Finalising startup...". Tested on Windows Sandbox (Windows 10).

![image](https://github.com/user-attachments/assets/47f5b2a3-babd-4ea5-a9c7-ed7bfc4ae44a)

**Comment by topoldo** [comment]
Hi.
As I promised, I installed Stirling-PDF (client for Windows v. 0.36.6) in another PC (maybe I can try in 3 other PCs in next days).

The PC is a Notebook HP Elite 8440p (2009) with Win11 Pro installed.
Its data (CPU, RAM etc...) are reported here:
![Immagine 2025-01-10 192442](https://github.com/user-attachments/assets/99db45f2-7f18-42c5-adba-366ed1fe52ad)

When I perform the installation in this PC what happens is the following:
1. the installation (run as admin ) is slow (it's an old machine with only 8GB of RAM!), it stops a little bit to 90%, then go on to the **END**.
2. When I relaunched the program, it opened but what I got was a _blank screen_.
3. I checked for the Task Manager and what I got was:
![Immagine 2025-01-10 192611](https://github.com/user-attachments/assets/f175f092-0c90-42af-a9c2-1e0e6c7e77ae)
so it seems there wasn't a problem of CPU/RAM.
4. When I killed the process(es) Task Manager was this:
![Immagine 2025-01-10 192828](https://github.com/user-attachments/assets/c0961aef-d152-4018-90f6-19f13d1d1d5f)

The full log from the first installation to the killing of the second launch is reported here:
[info.log](https://github.com/user-attachments/files/18385744/info.log)

IMHO it could be important to check also the version of jdk used.
In my case I'm using: **jdk-21.0.5_x64_bin**, both in this PC and in the other one I used previously.

After this installation I got a flash of inspiration seeing what happened.
That is: even when I performed the _very first_ installation of Stirling-PDF 0.36.6 on my main PC, _I was successfull_.
At that time I did not give importance to that fact. Then, in the following installations I started to have freezing at 90%.
So it seems there is something like an "imprinting" created during the first installation that prevents following installations/executions.

My question: when we uninstall this version of Stirling-PDF, we really **delete** any key/stuff from the Registry?
Is it stupid thinking that something stays there and creates some problems later?

Regards,
Topoldo

**Comment by djc0739** [comment]
Trying to download test version 3 to give it another try, but I am receiving a page not found error. Is it no longer available?

**Comment by Frooodle** [comment]
Correct https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.36.6/Stirling-PDF-test4.exe

**Comment by djc0739** [comment]
Tried test 4 installation with the same result. I tried stopping task, rebooting, and started app with the same result. Get to 90% and freezes. I am attaching the log.

I did notice that the app is attempting to use port 8080, which I am already using for something else.
[info.log](https://github.com/user-attachments/files/18387228/info.log)

**Comment by MetherFxcker** [comment]
您好，我是一名中国用户，我下载了Stirling-PDF-test4.exe文件并完成安装，但仍然在加载至90%时停止不动，无法进入应用，请问是否可以帮忙解决？

**Comment by Monoxide8969** [comment]
I too have the same issue with test4. Windows 11

**Comment by bret-miller** [comment]
I have Windows 11 24H2 as well. I downloaded test4, ran the installer as administrator, then ran Stirling-PDF as administrator. It "Finalising startup..." and sitting there at 90%.
[info.log](https://github.com/user-attachments/files/18397564/info.log)
[debug.log](https://github.com/user-attachments/files/18397575/debug.log)

After looking at debug.log, it said the cache path was invalid, so I created the cache directory shown and ran it again. Now it says:
"The cache_path directory (C:\Users\bret.miller\AppData\Roaming\Stirling-PDF\clientWebUI\cache) is not a child of the root_cache_path directory (C:\Users\bret.miller\AppData\Roaming\Stirling-PDF\clientWebUI\root_cache)"

But I don't a cache_path setting anywhere I can change. Just trying to see if these errors have anything to do with the app hanging at 90%.
[debug2.log](https://github.com/user-attachments/files/18397688/debug2.log)

**Comment by Strongly7402** [comment]
Tested on v0.37.0. Same issue again. I think issue can be related to Windows Firewall? I tested installation in Windows Sandbox. After a fresh install and first launch where it downloads bunch of fines, I get a Windows Security alert right when the launch gets stuck at 90%.  

![image](https://github.com/user-attachments/assets/49fdc9c9-3c6e-4159-a9eb-406dca30b240)

**Comment by topoldo** [comment]
Installing v. 0.37. Same result of 0.36.6. I also tried to disable Windows Firewall but freezing persists...
Topoldo

**Comment by Flithor** [comment]
For those waiting for the fix:
Even though if stuck on 90%, you still can use this tool by open http://localhost:8080/ in browser.
(or use the server version)

**Comment by yzsuai** [comment]
I also encountered the same problem

**Comment by deefrawley** [comment]
I have the same issue, Win 10, latest updates, Intel i5-8600k, 16GB DDR4 RAM but I have found a really odd fix for it.
Every time I start Systernals Process Explorer (procexp64.exe) the startup completes and the UI loads. No matter how long the 90% dialog box is on screen, 3-4 seconds after Process Explorer starts, the UI comes up (and I don't have to press or click anything).

**Comment by Frooodle** [comment]
Interesting this helps a lot! Thankyou

**Comment by liuyongshun2** [comment]
I have the same issue, win11 Intel(R) Core(TM) Ultra 7 155H，32GB RAM, Stirling PDF version 0.39.0. I try to uninstall or disable Windows Firewall, but nothing. so I just use localhost:8080. Is there any other way?

**Comment by KDPryor** [comment]
> I have the same issue, Win 10, latest updates, Intel i5-8600k, 16GB DDR4 RAM but I have found a really odd fix for it. Every time I start Systernals Process Explorer (procexp64.exe) the startup completes and the UI loads. No matter how long the 90% dialog box is on screen, 3-4 seconds after Process Explorer starts, the UI comes up (and I don't have to press or click anything).

Strangely enough that worked for me too. I'm on Windows 11, version 22H2.

**Comment by frankmichaels** [comment]
> I have the same issue, Win 10, latest updates, Intel i5-8600k, 16GB DDR4 RAM but I have found a really odd fix for it. Every time I start Systernals Process Explorer (procexp64.exe) the startup completes and the UI loads. No matter how long the 90% dialog box is on screen, 3-4 seconds after Process Explorer starts, the UI comes up (and I don't have to press or click anything).

I followed deefrawley's suggestion, 
The startup wa stuck at 90%
Installed "Systernals Process Explorer (procexp64.exe) " while it was stuck at 90%
The moment Process explorer started, Sterling PDF interface opened. 
Well, I have to repeat the process every time I need to open Sterling PDF.
Good app though. 
Thanks to the creators

**Comment by sjjsbsj** [comment]
encountered the same problem, stuck in 90%. Here's my log file: 

![Image](https://github.com/user-attachments/assets/c3796183-804f-493d-b6ab-cb9a9f2d39b6)

**Comment by frankmichaels** [comment]
Have you tried deefrawley's suggestion,
Installed "Systernals Process Explorer (procexp64.exe) " while it was stuck at 90%
The moment Process explorer started, Sterling PDF interface opened.
Well, I have to repeat the process every time I need to open Sterling PDF.

**Comment by Frooodle** [comment]
It actually seems to be fixed when any file browser opens for instance explorer.exe (windows file browser)
Good news is we have a 'fix' coming next release!

**Comment by frankmichaels** [comment]
> It actually seems to be fixed when any file browser opens for instance explorer.exe (windows file browser) Good news is we have a 'fix' coming next release!

Thank you Froodle for ur efforts

**Comment by ChenChenGith** [comment]
> It actually seems to be fixed when any file browser opens for instance explorer.exe (windows file browser) Good news is we have a 'fix' coming next release!

Thank you for your work. Hope for new version!

**Comment by KDPryor** [comment]
> It actually seems to be fixed when any file browser opens for instance explorer.exe (windows file browser) Good news is we have a 'fix' coming next release!

Great news! Thank you for this great tool.

**Comment by Frooodle** [comment]
Alright... lets see...
https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.42.0/Stirling-PDF-win-installer.exe

**Comment by Frooodle** [comment]
"It worked on my machine"

**Comment by KDPryor** [comment]
> Alright... lets see... https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.42.0/Stirling-PDF-win-installer.exe
> 
> Fingers crossed

That didn't help unfortunately. It still sticks at 90% and never gets past that unless I open Process Explorer like before.

**Comment by Frooodle** [comment]
Weird in my testing after 4 seconds it should bypass and resolve.... Damm.. presume same for everyone else?

**Comment by SystemlessDev** [comment]
Hey! 
Seems like it's still a problem. Oddly enough I don't experience this issue on my personal computer. Just my work computer. 
Here's a copy of the startup logs on the problematic machine. 

[info.log](https://github.com/user-attachments/files/18852049/info.log)

**Comment by Frooodle** [comment]
Thanks!

**Comment by KDPryor** [comment]
[info.log](https://github.com/user-attachments/files/18852156/info.log)

> Weird in my testing after 4 seconds it should bypass and resolve.... Damm.. presume same for everyone else?
> 
> Thanks for testing any new logs in appdata/Stirling pdf/logs ? Should be some new startup logs to help debug too

Forgot to include the log in my last post. Here it is:

**Comment by djc0739** [comment]
The new version did not work for me. The log is attached.

[info.log](https://github.com/user-attachments/files/18854027/info.log)

**Comment by yichifauzi** [comment]
Still stuck in 90% for me too (Windows 11)

[info.log](https://github.com/user-attachments/files/18857259/info.log)

**Comment by topoldo** [comment]
Still stuck at 90% for me, too (Windows 11 Pro 24H2)
The log is attached

[info.log](https://github.com/user-attachments/files/18862575/info.log)

**Comment by gaoqian310** [comment]
I'm using a laptop with an external monitor, when stuck at 90%, closing the lid brings it to finish the initialize process.

If I disconnect the external monitor, closing the lid will do nothing.

[info.log](https://github.com/user-attachments/files/18869722/info.log)

**Comment by kidneywiki** [comment]
Getting this issue on a brand new device with fresh Windows 11 x64 install. Used the latest Stirling release with the "fix" in it. IIRC, all defaults used during setup. Says it is stuck on 90% but I checked the logs and saw that it is working on localhost. Screenshot and log attached.

[info.log](https://github.com/user-attachments/files/18895765/info.log)

![Image](https://github.com/user-attachments/assets/07457c7c-c519-43d2-884c-4428b1f21505)

**Comment by Ludy87** [comment]
@kidneywiki Is it an Intel CPU?

**Comment by kidneywiki** [comment]
> @kidneywiki Is it an Intel CPU?

Correct

**Comment by cx1tp** [comment]
I'd like to provide some additional information based on my log files:

1. After setting up the application, the interface was blank.
2. When starting another instance, the log file indicated that port 8080 was already in use. Upon checking, I found that Stirling PDF was occupying port 8080, possibly due to an attempt to start another instance.
3. I terminated the process and restarted Stirling PDF, but the interface remained blank.
4. However, when accessing `http://localhost:8080` through a web browser, everything functioned correctly.

**Comment by djc0739** [comment]
"When starting another instance, the log file indicated that port 8080 was already in use. Upon checking, I found that Stirling PDF was occupying port 8080, possibly due to an attempt to start another instance."

While I freeze at 90% during the installation, in my case I am already using port 8080 for another application.

**Comment by Thickness6871** [comment]
I could install the program, however, I have a similar problem, when i try to open it. This is weird.
Windows 11 Pro 23H2

[info.log](https://github.com/user-attachments/files/18924207/info.log)

I've opened it only once, then I wanted to install Tesseract, but StirlingPDF never opened again.
Now it stops in the same place - Finalising startup - 90% 
In task manage I see 2 processes for stirling ( in details  Stirling-PDF.exe and svchost)
After some time, two Java Chromium Embedded Framework Helper jcef_helper.exe processes also appear.

I tried to reinstall it without any result.

While trying to open this debug.log appears in Stirling installation folder:
[debug.log](https://github.com/user-attachments/files/18924111/debug.log)

**Comment by headspinning** [comment]
So I was having the 90% problem. Info.log stopped a listening port :8080. I uninstalled. I installed this https://github.com/Stirling-Tools/Stirling-PDF/releases/download/v0.36.5/Stirling-PDF-test.exe
and a big difference is that the default install directory is c:\users\*username*\AppData\Local\Stirling-PDF\  . The previous install was at C:\Program Files\Stirling-PDF\

This computer has
c:\users\*username*\AppData\Local\Stirling-PDF\ has these files and folders:

"c\Users\username\AppData\Local\Stirling-PDF\debug.log"
"c\Users\username\AppData\Local\Stirling-PDF\Stirling-PDF.exe"
"c\Users\username\AppData\Local\Stirling-PDF\app"
"c\Users\username\AppData\Local\Stirling-PDF\configs"
"c\Users\username\AppData\Local\Stirling-PDF\customFiles"
"c\Users\username\AppData\Local\Stirling-PDF\jcef-bundle"
"c\Users\username\AppData\Local\Stirling-PDF\logs"
"c\Users\username\AppData\Local\Stirling-PDF\pipeline"
"c\Users\username\AppData\Local\Stirling-PDF\runtime"

and at
"C:\Users\username\AppData\Roaming\Stirling-PDF\logs"
"C:\Users\username\AppData\Roaming\Stirling-PDF\pipeline"
"C:\Users\username\AppData\Roaming\Stirling-PDF\clientWebUI"
"C:\Users\username\AppData\Roaming\Stirling-PDF\configs"
"C:\Users\username\AppData\Roaming\Stirling-PDF\customFiles"

--------------------
I have another computer where Stirling-PDF.exe is installed at  C:\Program Files\Stirling-PDF\ and working fine, but only has these files and folders:
"C:\Program Files\Stirling-PDF\Stirling-PDF.exe"
"C:\Program Files\Stirling-PDF\app"
"C:\Program Files\Stirling-PDF\runtime"

The rest at
"C:\Users\username\AppData\Roaming\Stirling-PDF\logs"
"C:\Users\username\AppData\Roaming\Stirling-PDF\pipeline"
"C:\Users\username\AppData\Roaming\Stirling-PDF\clientWebUI"
"C:\Users\username\AppData\Roaming\Stirling-PDF\configs"
"C:\Users\username\AppData\Roaming\Stirling-PDF\customFiles"

I hope that helps. Note: it could also be uninstall and reinstall. I may play around later with the install, but going on vacation now.

**Comment by linpxing579** [comment]
I also encountered this problem when installing version 0.43.2. Is there a solution?

**Comment by rishabhkrmahato** [comment]
I can confirm this issue as well. Although the app is accessible at localhost:8080, a dialog box appears and remains stuck at 90%.

![Image](https://github.com/user-attachments/assets/e989d1a6-a74c-4f71-960a-8c6ef4832f53)

**Comment by Frooodle** [comment]
One user recently said they had a fix for this., this is included in 0.44.0 release, can someone confirm if it works for them?

**Comment by MathisP75** [comment]
I can confirm it is fixed for me as of release 0.44.0

**Comment by Frooodle** [comment]
Awesome! Will wait for one more confirmation then close ticket!

**Comment by djc0739** [comment]
I can confirm it worked for me on a Win 11 machine. I won't be able to try my Win 10 machine until tomorrow.
Thanks for keeping at this!

**Comment by deefrawley** [comment]
Hi, 0.44.0 working for me now.

**Comment by yxf10093** [comment]
I'm also having this problem, but it's at 80%, I try to uninstall and change to the test mentioned above, but then it's still stuck at 80%, I only keep the log of the test

**Comment by Frooodle** [comment]
Issue is resolved please don't use any test file but only the latest release 
If you still have issues then please follow-up in new ticket

**Comment by yxf10093** [comment]
Okay，thank you

**Comment by hjtoh** [comment]
the problem still exist on the latest version, it exit unexpectedly at 80% initialising. it's really frustrating since the issue stays unresolved all the time.

**Comment by Frooodle** [comment]
Existing at 80% is a very different issue to it freezing at 80% and the report of this issue has vastly died down and resolved for many 

Are you able to provide your log file from appdata/Stirling pdf

**Comment by jvdstoel-home** [comment]
Hello, I downloaded Windows (11) installer on 18/8. Installer fileversion is 1.2.0. Filename is Stirling-PDF-win-installer.exe. Download file location: https://files.stirlingpdf.com/win-installer.exe  Same problem as described in the first debug description.

[info-2025-08-18.log](https://github.com/user-attachments/files/21856481/info-2025-08-18.log)

[info.log](https://github.com/user-attachments/files/21856486/info.log)

Installation freezes at 90%. The popup displays the message:

Initializing Stirling-PDF
Stirling-PDF
Finalising startup...
90%
